db/schema/Final_schemas.sql

```diff
@@ -1,6 +1,8 @@
 -- RUN THIS file
--- CREATE table users
+-- Grant privliges to database user
+ALTER USER labber WITH SUPERUSER;
```

Comment:
When querying into the database I was running into permission issues, so I resolved that issue by giving "labber" superuser permissions